man/docker-build.1.md

```diff
@@ -235,6 +235,12 @@ If you do not provide a version tag then Docker will assign `latest`:
 
 When you list the images, the image above will have the tag `latest`.
 
+Sometimes you may want to update the `latest` tag to point to the
+newly built image as well as keeping a long term tag to reference a specific
+version. This can be done by :
+
+    docker build -t whenry/fedora-jboss:v2.1 -t whenry/fedora-jboss:latest .
+
```

Comment:
I actually think that's looking quite good; it fits nicely with the way the other examples on this page are written. 
I'd replace "you may want to", with "you want to".

Perhaps replacing "This can be done by" with:

```Markdown
For example, to tag an image both as `whenry/fedora-jboss:v2.1`
and `whenry/fedora-jboss:latest`, use the following: 
```

I'm sure @moxiegirl can improve on that, but we'll see :+1: